dpctl/tensor/libtensor/include/kernels/accumulators.hpp

```diff
@@ -84,9 +84,18 @@ template <typename srcTy, typename dstTy> struct CastTransformer
     }
 };
 
+template <typename ScanOpT, typename T> struct needs_workaround
+{
+    // work-around needed due to crash in JITing on CPU
+    static constexpr bool value =
+        std::is_same_v<ScanOpT, sycl::logical_or<T>> ||
```

Comment:
There are `IsSyclLogicalOr` and `IsSyclLogicalAnd` declared in `dpctl/tensor/libtensor/include/utils/sycl_utils.hpp`.
Should they be used here?